taxi/test/test_models.py

```diff
@@ -0,0 +1,59 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.forms import DriverCreationForm
+from taxi.models import Manufacturer, Car
+
+
+class TestCreateAndDeleteModels(TestCase):

```

Comment:
no need to test build in methods here,  u can check some craft methods like str etc.